script.js

```diff
@@ -12,4 +12,137 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+  liveDemo: 'https://george827.github.io/My-Portfolio/',
+  sourceCode: 'https://github.com/george827/My-Portfolio',
+  title1: 'Tonic',
+  title2: 'Tonic',
+},
+{
+  description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+  description2: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+  img1: './img/Snapshoot Portfolio4.png',
+  img2: './img/Snapshoot Portfolio4.png',
+  technologies: ['Html', 'Css', 'Javascript'],
+  liveDemo: 'https://george827.github.io/My-Portfolio/',
+  sourceCode: 'https://github.com/george827/My-Portfolio',
+  title1: 'Multi-Post Stories',
+  title2: 'Multi-Post Stories',
+},
+{
+  description: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+  description2: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+  img1: './img/Snapshoot-Portfolio-DESK3.png',
+  img2: './img/Snapshoot-Portfolio-DESK3.png',
+  technologies: ['Html', 'Css', 'Javascript'],
+  liveDemo: 'https://george827.github.io/My-Portfolio/',
+  sourceCode: 'https://github.com/george827/My-Portfolio',
+  title1: 'Facebook 360',
+  title2: 'Facebook 360',
+},
+{
+  description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+  description2: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+  img1: './img/Snapshoot-Portfolio-DESK4.png',
+  img2: './img/Snapshoot-Portfolio-DESK4.png',
+  technologies: ['Html', 'Css', 'Javascript'],
+  liveDemo: 'https://george827.github.io/My-Portfolio/',
+  sourceCode: 'https://github.com/george827/My-Portfolio',
+  title1: 'Uber Navigation',
+  title2: 'Uber Navigation',
+},
+];
+
+const section = document.querySelector('.works_section');
+const popup = document.querySelector('.popup-window');
+
+project.forEach((projects) => {
+  const article = document.createElement('article');
+  const divImg = document.createElement('div');
+  const img = document.createElement('img');
+  const elements = document.createElement('div');
+  const button = document.createElement('button');
+  img.src = `${projects.img1}`;
+  article.classList.add('card1');
+  button.classList.add('btn-main');
+  elements.classList.add('elements1');
+  button.innerText = 'See Project';
+  img.classList.add('carImg');
+
+  elements.innerHTML += `
+  <div class="sideSection">
+      <h2 class="head-one">${projects.title1}</h2>
+      <ul class="highlights">
+          <li class="canop">CANOPY</li>
+          <li> <img class="counter" src="img/counter.png" alt="counter">  </li>
+          <li>Back End Dev</li>
```

Comment:
- [ ] Kindly make sure you set the proper path to your images. 

`hint: You can consider adding ./image/...`
![image](https://user-images.githubusercontent.com/97100091/196738621-99fca492-790b-4cbe-898b-d75bd73d55cb.png)